ocs_ci/ocs/node.py

```diff
@@ -332,8 +333,11 @@ def add_new_node_and_label_upi(node_type, num_nodes, mark_for_ocs_label=True):
         status=constants.NODE_READY
     )
 
+    new_spun_nodes = list(set(nodes_after_exp) - set(initial_nodes))
+    if node_type == constants.RHEL_OS:
+        assert set_selinux_permissions(workers=new_spun_nodes), "Adding workaround for the issue #1151 failed"
```

Comment:
no need to assert once you remove `return True` from `set_selinux_permissions()`